package.json

```diff
@@ -66,6 +66,9 @@
     ],
     "*.robot": [
       "python3 -m pipenv run robotidy --config tests/robot-tests/robotidy.toml"
+    ],
+    "*.py": [
```

Comment:
Do we also want `format:python` and `format:python:check` scripts, similar to what we have to robotidy?